config/routes.rb

```diff
@@ -3,6 +3,8 @@
   devise_for :users
   post 'create_booking', controller: :dashboard, action: :create_booking
   root controller: :dashboard, action: :index
+  resources :pages, only: [:show]
```

Comment:
According to the docs this is the way this route should be configured
```
get "/pages/*id" => 'pages#show', as: :page, format: false
```
but it feels a bit outdated to me, 
However, `resource` and  `resources` feels more for CRUD actions. 
Not sure about this though.